src/main/java/com/github/kaktushose/jda/commands/internal/YamlLoader.java

```diff
@@ -22,15 +25,14 @@
  * @since 1.0.0
  */
 public final class YamlLoader {
-
     /**
      * Loads a yaml file and creates a {@link CommandSettings} object from it.
      *
      * @param url The URL where the yaml file is located
      * @return The {@link CommandSettings} object
      * @throws IOException If an I/O error occurs
      */
-    public static CommandSettings load(URL url) throws IOException {
+    public static CommandSettings load(URL url) throws IOException, ClassNotFoundException {
```

Comment:
same as for JDACommandsBuilder. `ClassNotFoundException` has nothing to do with settings files